CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 - Added shortcodes for [fork-awesome icons](https://forkaweso.me/Fork-Awesome/icons/) (e.g. `:fa-picture-o:`)
 - The code button now adds code fences even if the user selected nothing beforehand
 - Code blocks with 'csv' as language render as tables.
+- Code blocks have a 'Copy Code' button.
```

Comment:
Mention the copy-to-clipboard functionality. An external user reading this might be irritated where the code gets copied to.

Reply:
done